dockerunit-consul/src/main/java/com/github/qzagarese/dockerunit/discovery/consul/ConsulDescriptor.java

```diff
@@ -37,18 +40,32 @@ public CreateContainerCmd setup(CreateContainerCmd cmd) {
             bindings = new Ports();
         }
         
-        int dnsBridgePort = Integer.parseInt(System.getProperty(CONSUL_DNS_PORT_BRIDGE_BINDING, 
-                CONSUL_DNS_PORT_BRIDGE_BINDING_DEFAULT));
+        Optional<String> disableDnsFlag = Optional.ofNullable(System.getProperty(CONSUL_DNS_OFF_PROPERTY, CONSUL_DNS_OFF_DEFAULT));
```

Comment:
Why do we need to wrap this in an Optional? Won't the 
```
System.getProperty(CONSUL_DNS_OFF_PROPERTY, CONSUL_DNS_OFF_DEFAULT)
``` 
always return a non-null value?

Reply:
If the flag is not passed at all, it returns null.

Reply:
Humm `System.getProperty` returns `CONSUL_DNS_OFF_DEFAULT` if the flag is not set so it never is null, it's either what `CONSUL_DNS_OFF_PROPERTY` env var has been set to or whatever the value of `CONSUL_DNS_OFF_DEFAULT` is

Reply:
I've thought about and I believe you are right.
I'm going to change this flag to `consul.dns.enabled` with a default to `true`.
Thank you